packages/atlas/src/api/queries/nfts.graphql

```diff
@@ -40,8 +40,18 @@ query GetNftsConnection(
   }
 }
 
-query GetFeaturedNfts {
-  ownedNfts(limit: 5, orderBy: [createdAt_DESC]) {
+query GetFeaturedNftsVideos {
+  ownedNfts(
+    limit: 5
+    orderBy: [createdAt_DESC]
+    where: {
+      isFeatured_eq: true
+      transactionalStatus: {
+        isTypeOf_in: ["TransactionalStatusAuction", "TransactionalStatusBuyNow"]
+        auction: { isCompleted_eq: false }
```

Comment:
also, why you're not using `useFeaturedNfts` query here?

Reply:
> also, why you're not using `useFeaturedNfts` query here?

What do you mean here? I'm still using it, I only renamed it. I think the `FeaturedNftsVideos` name is more suitable because it's fetching NFTs together with videos data(the query is used for NFTs video carousel in marketplace)